src/cloud/components/molecules/ContentManager/ContentManagerToolbar.tsx

```diff
@@ -411,96 +441,101 @@ const ContentManagerToolbar = ({
-              onClick={(event) => {
-                openContextModal(
-                  event,
-                  <DocLabelSelectionModal
-                    selectedTags={selectedDocumentsCommonValues.tags}
-                    sendTags={sendTags}
-                  />,
-                  {
-                    alignment: 'top-left',
-                    width: 300,
-                  }
-                )
-              }}
-              disabled={selectedDocsAreUpdating}
-              spinning={sending === BulkActions.label}
-            >
-              {translate(lngKeys.GeneralLabels)}
-              {selectedDocumentsCommonValues.tags.length > 0
-                ? ` (${selectedDocumentsCommonValues.tags.length})`
-                : null}
-            </LoadingButton>
-            <DocAssigneeSelect
-              isLoading={sending === BulkActions.assignees}
-              disabled={selectedDocsAreUpdating}
-              defaultValue={selectedDocumentsCommonValues.assignees}
-              readOnly={selectedDocsAreUpdating}
-              update={sendUpdateAssignees}
-              popupAlignment='top-left'
-            />
-            <DocStatusSelect
-              status={selectedDocumentsCommonValues.status}
-              sending={sending === BulkActions.status}
-              onStatusChange={sendUpdateStatus}
-              disabled={selectedDocsAreUpdating}
-              isReadOnly={selectedDocsAreUpdating}
-              onClick={(event) => {
-                openContextModal(
-                  event,
-                  <ContentManagerToolbarStatusPopup
-                    onStatusChange={sendUpdateStatus}
-                  />,
-                  {
-                    alignment: 'top-left',
-                    width: 300,
-                  }
-                )
-              }}
-            />
-            <DocDueDateSelect
-              dueDate={selectedDocumentsCommonValues.dueDate}
-              isReadOnly={selectedDocsAreUpdating}
-              sending={sending === BulkActions.duedate}
-              shortenedLabel={true}
-              onDueDateChange={sendUpdateDocDueDate}
-            />
-          </>
-        )}
-        <LoadingButton
-          className='cm__tool'
-          variant='bordered'
-          iconPath={mdiFolderMoveOutline}
-          onClick={openMoveForm}
-          disabled={selectedDocsAreUpdating || selectedFoldersAreUpdating}
-          spinning={sending === BulkActions.move}
-        >
-          {translate(lngKeys.GeneralMoveVerb)}
-        </LoadingButton>
-        <LoadingButton
-          className='cm__tool'
-          variant='bordered'
-          iconPath={mdiTrashCanOutline}
-          onClick={bulkDeleteCallback}
-          disabled={selectedDocsAreUpdating || selectedFoldersAreUpdating}
-          spinning={sending === BulkActions.delete}
-        >
-          {translate(lngKeys.GeneralDelete)}
-        </LoadingButton>
-      </div>
-    </Container>
+    <>
```

Comment:
Needed because container has transform and to break out of HTML structure it won't work with fixed positioning, see here: https://stackoverflow.com/questions/2637058/positions-fixed-doesnt-work-when-using-webkit-transform